run.20230215.NGD_v3atm.F2010.P10.sh

```diff
@@ -0,0 +1,657 @@
+         'C2H6     -> \$DIN_LOC_ROOT/atm/cam/chem/trop_mozart_aero/emis/DECK_ne30/v3-f2010-1/emissions-cmip6_e3sm_C2H6_surface_2010_1.9x2.5_c20210323.nc', 
+         'C3H8     -> \$DIN_LOC_ROOT/atm/cam/chem/trop_mozart_aero/emis/DECK_ne30/v3-f2010-1/emissions-cmip6_e3sm_C3H8_surface_2010_1.9x2.5_c20210323.nc',
+         'CH2O     -> \$DIN_LOC_ROOT/atm/cam/chem/trop_mozart_aero/emis/DECK_ne30/v3-f2010-1/emissions-cmip6_e3sm_CH2O_surface_2010_1.9x2.5_c20210323.nc',
+         'CH3CHO   -> \$DIN_LOC_ROOT/atm/cam/chem/trop_mozart_aero/emis/DECK_ne30/v3-f2010-1/emissions-cmip6_e3sm_CH3CHO_surface_2010_1.9x2.5_c20210323.nc',
+         'CH3COCH3 -> \$DIN_LOC_ROOT/atm/cam/chem/trop_mozart_aero/emis/DECK_ne30/v3-f2010-1/emissions-cmip6_e3sm_CH3COCH3_surface_2010_1.9x2.5_c20210323.nc',
+         'CO       -> \$DIN_LOC_ROOT/atm/cam/chem/trop_mozart_aero/emis/DECK_ne30/v3-f2010-1/emissions-cmip6_e3sm_CO_surface_2010_1.9x2.5_c20210323.nc',       
+         'ISOP     -> \$DIN_LOC_ROOT/atm/cam/chem/trop_mozart_aero/emis/DECK_ne30/v3-f2010-1/emissions-cmip6_e3sm_ISOP_surface_2010_1.9x2.5_c20210323.nc',
+         'C10H16   -> \$DIN_LOC_ROOT/atm/cam/chem/trop_mozart_aero/emis/DECK_ne30/v3-f2010-1/emissions-cmip6_e3sm_MTERP_surface_2010_1.9x2.5_c20220426.nc',         
+         'SOAG0     -> \$DIN_LOC_ROOT/atm/cam/chem/trop_mozart_aero/emis/DECK_ne30/v3-f2010-1/SOC_CEDS_2010_QZRNGDAMIP_F4.nc',
+         'NO        -> \$DIN_LOC_ROOT/atm/cam/chem/trop_mozart_aero/emis/DECK_ne30/v3-f2010-1/emissions-cmip6_e3sm_NO_surface_2010_1.9x2.5_c20220425.nc',    
+         'DMS       -> \$DIN_LOC_ROOT/atm/cam/chem/trop_mozart_aero/emis/DMSflux.2010.1deg_latlon_conserv.POPmonthlyClimFromACES4BGC_c20190220.nc',
+         'SO2       -> \$DIN_LOC_ROOT/atm/cam/chem/trop_mozart_aero/emis/DECK_ne30/cmip6_mam4_so2_surf_1x1_2010_clim_c20190821.nc',
+         'bc_a4     -> \$DIN_LOC_ROOT/atm/cam/chem/trop_mozart_aero/emis/DECK_ne30/cmip6_mam4_bc_a4_surf_1x1_2010_clim_c20190821.nc',
+         'num_a1    -> \$DIN_LOC_ROOT/atm/cam/chem/trop_mozart_aero/emis/DECK_ne30/cmip6_mam4_num_a1_surf_1x1_2010_clim_c20190821.nc',
+         'num_a2    -> \$DIN_LOC_ROOT/atm/cam/chem/trop_mozart_aero/emis/DECK_ne30/cmip6_mam4_num_a2_surf_1x1_2010_clim_c20190821.nc',
+         'num_a4    -> \$DIN_LOC_ROOT/atm/cam/chem/trop_mozart_aero/emis/DECK_ne30/cmip6_mam4_num_a4_surf_1x1_2010_clim_c20190821.nc',
+         'pom_a4    -> \$DIN_LOC_ROOT/atm/cam/chem/trop_mozart_aero/emis/DECK_ne30/cmip6_mam4_pom_a4_surf_1x1_2010_clim_c20190821.nc',
+         'so4_a1    -> \$DIN_LOC_ROOT/atm/cam/chem/trop_mozart_aero/emis/DECK_ne30/cmip6_mam4_so4_a1_surf_1x1_2010_clim_c20190821.nc',
+         'so4_a2    -> \$DIN_LOC_ROOT/atm/cam/chem/trop_mozart_aero/emis/DECK_ne30/cmip6_mam4_so4_a2_surf_1x1_2010_clim_c20190821.nc'
+         'E90       -> \$DIN_LOC_ROOT/atm/cam/chem/trop_mozart_aero/emis/DECK_ne30/v3-f2010-1/emissions_E90_surface_2010_1.9x2.5_c20210408.nc'
+ srf_emis_type          = 'CYCLICAL'
```

Comment:
also need to change SOAG0 emissions here for both ext and srf

Reply:
This script is removed.